.github/workflows/coq-action.yml

```diff
@@ -16,9 +16,7 @@ jobs:
       matrix:
         image:
           - 'coqorg/coq:dev'
-          - 'coqorg/coq:8.12'
```

Comment:
Did you intend to drop support for 8.12?

Reply:
Well, I *am* testing 8.12: `mathcomp/mathcomp:1.11.0-coq-8.12`

Reply:
But I am puzzled/annoyed that `coq-elpi` fails to build with the standard settings of the coq:dev image. I cannot reproduce this build failure in my local dev switch, but that uses ocaml 4.09 while the dev image seems to default to the 4.05 switch for ocaml.